configs/usr/lib/cgi-bin/download_rootfs.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+MOUNT_DIR=$(mktemp -d -t "WB-RFS.XXXXXXX")
+SERIAL=$(cat "/var/lib/wirenboard/short_sn.conf")
+ROOTFS_DEV=$(mount | grep "on / type ext4" | awk '{print $1}')
+
+function cleanup()
+{
+    sudo umount -f "$MOUNT_DIR" || true
+    rm -r "$MOUNT_DIR"
+}
+
+trap cleanup EXIT
+
+sudo mount "$ROOTFS_DEV" "$MOUNT_DIR"
+
+if [ -z "$(ls -A $MOUNT_DIR)" ]; then
+    echo "Status: 500"
+    echo ""
+    echo "Error mounting rootfs"
+else
+    echo "Status: 200"
+    echo "Content-Disposition: attachment; filename=\"rootfs_${SERIAL}.tar.gz\""
+    echo "Content-Type: application/gzip"
+    echo ""
+    sudo tar -czf - "$MOUNT_DIR"
```

Comment:
Попробовал скачать rootfs с WB7, скорость через провод к середине процесса дошла до 900 КБ/с. Если заменить это на:

```suggestion
    sudo tar -cf - "$MOUNT_DIR" | pigz
```

процесс идёт раза в 3-4 быстрее (просто потому что pigz использует все ядра процессора, а у WB7 их четыре).

Для этого pigz должен быть в зависимостях, естественно

Reply:
Всё причесал, потестил -- генериться стало в 3-4 раза быстрее, лишней иерархии в архиве нет.